src/server/_query.py

```diff
@@ -499,7 +546,14 @@ def to_asc(v: Union[str, bool]) -> str:
                 return "DESC"
             return cast(str, v)
 
-        args_order = [f"{self.alias}.{k} ASC" for k in args]
+        # Use the column name without their table name for the Order By clause since Union All is used
+        args_order = []
+        for i in range(len(args)):
+            # avoid conflict with the reserved word
+            if args[i] == 'signal':
+                args_order.append(f"`signal` ASC")
+            else:
+                args_order.append(f"{args[i]} ASC")
         kw_order = [f"{self.alias}.{k} {to_asc(v)}" for k, v in kwargs.items()]
         self.order = args_order + kw_order
```

Comment:
looks like we'll end up with a mix of columns in ORDER BY, some without a table prefix, and some with, depending on whether they come from *args or **kwargs when `set_order` is called. Is that intended?

Reply:
I changed here to all without the table prefix.